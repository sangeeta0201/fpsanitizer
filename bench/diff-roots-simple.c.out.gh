Output @ diff-roots-simple.c:8 in main (addr 400695)
61.959049 bits average error
61.959049 bits max error
Aggregated over 1 instances
Influenced by erroneous expression:

    (FPCore ()
          (- (sqrt (+ 1.000000 1.000000e16)) (sqrt 1.000000e16)))
   diff-roots-simple.c:7 in main (addr 40066D)
   61.959049 bits average error
   61.959049 bits max error
   61.959049 bits average local error
   61.959049 bits max local error
   Aggregated over 1 instances

