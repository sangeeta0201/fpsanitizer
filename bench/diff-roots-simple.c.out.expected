(output
  (argIdx 0)
  (function "main")
  (filename "diff-roots-simple.c")
  (line-num 8)
  (instr-addr 4006B4)
  (avg-error 61.959049)
  (max-error 61.959049)
  (num-calls 1)
  (influences
    (
    (
     (expr
       (FPCore ()
          (- (sqrt (+ 1.000000 1.000000e16)) (sqrt 1.000000e16))))
     (var-problematic-ranges)
     (example problematic input ())
     (function "main")
     (filename "diff-roots-simple.c")
     (line-num 7)
     (instr-addr 40068C)
     (avg-error 61.959049)
     (max-error 61.959049)
     (avg-local-error 61.959049)
     (max-local-error 61.959049)
     (num-calls 1))
    )
  )
)
